NextToMeAPI/NextToMe.Services/EmailService.cs

```diff
@@ -18,7 +18,8 @@ namespace NextToMe.Services
 {
     public class EmailService : IEmailService
     {
-        private const string _emailTemplateFile = "emailTemplates/invitation.html";
+        private const string _emailConfirmTemplateFile = "../NextToMe.Services/emailTemplates/invitation.html";
+        private const string _emailResetPasswordTemplateFile = "../NextToMe.Services/emailTemplates/reseting.html";
```

Comment:
```suggestion
        private const string _emailConfirmTemplateFile = "emailTemplates/invitation.html";
        private const string _emailResetPasswordTemplateFile = "emailTemplates/reseting.html";
```
нам надо указывать путь в папке сборки. 
На машине, где запускается код, может отсутствовать "../NextToMe.Services/emailTemplates/" потому что это папка из структуры проекта разработки.
Но мы можем быть уверены, что "emailTemplates\reseting.html" и "emailTemplates\invitation.html" будут лежать в папке с бинарниками, потому что так сконфигурировано в csproj-е:

` <CopyToOutputDirectory>Always</CopyToOutputDirectory>`

https://github.com/frolowova/NextToMe/pull/19/files#diff-f372aaf135116a7e40b5c951fe7d73ebde50c31f2d61fbc6b95cfabd2cefdff8R28